client_plugin/drivers/shared/etcdops.go

```diff
@@ -255,9 +292,11 @@ func joinEtcdCluster(nodeAddr string, leaderAddr string, nodeID string) error {
 		"--initial-cluster-state", etcdClusterStateExisting,
 	}
 
+	// start the etcd cluster routine
```

Comment:
Why the comments is start the etcd cluster routine? I think it should be "join the etcd cluster"?